src/steps/roles/converters.ts

```diff
@@ -0,0 +1,53 @@
+import generateEntityKey from '../../utils/generateEntityKey';
+import { entities } from '../../constants';
+import { createIntegrationEntity } from '@jupiterone/integration-sdk-core';
+import { admin_directory_v1 } from 'googleapis';
+import Schema$Role = admin_directory_v1.Schema$Role;
+
+export type RolePrivilegeStrings = {
+  privilegeIds: string[];
+  privilegeNames: string[];
+};
+
+export function getRolePrivilegeStrings(
+  role: Schema$Role,
+): RolePrivilegeStrings {
+  return (role.rolePrivileges || []).reduce(
+    (privileges, role) => {
+      if (role.serviceId) privileges.privilegeIds.push(role.serviceId);
+      if (role.privilegeName)
+        privileges.privilegeNames.push(role.privilegeName);
+
+      return privileges;
+    },
+    { privilegeIds: [], privilegeNames: [] } as RolePrivilegeStrings,
```

Comment:
Just for consistent naming

```suggestion
    { privilegeServiceIds: [], privilegeNames: [] } as RolePrivilegeStrings,
```